tailwind.config.js

```diff
@@ -2,6 +2,13 @@
 export default {
 	content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
 	theme: {
+		colors: {
+			primary: '#225F33',
+			'light-red': '#BA6E6E',
+			secondary: 'rgba(217, 204, 159, 0.75)',
+			white: '#fff',
+			black: '#2D2D2D',
+		},
```

Comment:
put the `colors `object into `extend`.   if the colors object is not included within extend, it will overwrite the default Tailwind colors, which could lead to unexpected results. For example, if a color with the same name as a default Tailwind color is defined in the colors object, it will overwrite the default color, which could cause issues if the default color is relied upon in other parts of the application.

Reply:
done